src/operations/utils/areAllShapesTheSameType.js

```diff
@@ -0,0 +1,18 @@
+// list of supported geometries
```

Comment:
to me this does not belong in operations at all but in core/common reusable code, as it is not specific to operations at all

Reply:
i'm not even sure about the name. shouldn't this now be 'areAllGeometriesTheSame'? maybe this should be a util function as part of the geometry?

Reply:
'areAllGeometriesTheSame.js' has been moved to src/utils/areAllGeometriesTheSame.js